runner/collect_results.py

```diff
@@ -16,14 +19,18 @@ def dir_path(string):
 
 def main():
     parser = argparse.ArgumentParser()
-    parser.add_argument("src_path", type=dir_path)
+    parser.add_argument("src_path")
     parser.add_argument("out_file")
     parser.add_argument("--raw", dest="raw", action="store_true")
     args = parser.parse_args()
 
     # collect data from all runs
+    src_path = (
+        args.src_path if args.src_path != "latest"
```

Comment:
I think it might be cleaner to switch arguments (`out_file` first, then `src_path`) and make `src_path` optional. If it is not given, we use the latest run. Using `latest` as a magic path seems a bit risky.